etc/profile-a-l/audacity.profile

```diff
@@ -20,9 +20,15 @@ include disable-xdg.inc
 
 include whitelist-var-common.inc
 
+# Silence blacklist violation. See #5539.
+allow-debuggers
 ## Enabling App Armor appears to break some Fedora / Arch installs
 #apparmor
 caps.drop all
+# Add the below lines to your audacity.local if you need online plugins.
+#ignore net none
+#netfilter
+#protocol inet6
```

Comment:
Suggestion: Move these lines to the top of the profile (below
`include globals.local`), since they contain an `ignore` command and also to
make the list around them more readable.